### ISSUE TITLE
Time scale starts at hour 21st and finishes at the 20th hour in the Hourly Details vision

### ISSUE THREAD

**Issue by Joao31415** [amit9838/mousam#164]
Time scale starts at hour 21st and finishes at the 20th hour in the Hourly Details vision when the expected would be to start at 00hs and finish at 23hs.

![Screenshot_20241027_183521](https://github.com/user-attachments/assets/85c60447-2352-4869-b81e-380a32aff4cf)
![Screenshot_20241027_183536](https://github.com/user-attachments/assets/df5ec9ab-64df-4690-817d-04501d27cab9)

I believe that this may be a time zone issue as I am located in GMT-03.

**Comment by amit9838** [comment]
yes that's true it is due to the timezone. And that's a good suggestion we can start from 00:00 to 23:59

**Comment by womble75** [comment]
Would be good if the timezone is taken into account when trying to view a location outside your own. For instance I'm currently in GMT but if I want to look at a place located in GMT-05 the view still uses my current time as "Now" and shows the corresponding weather details instead of the local time and details at the location I want to view.
The background updates as I'd expect so while it is daylight here it is still dark in GMT-05 and the background reflects that, but the time highlighted and current weather details shown correspond to GMT.

**Comment by amit9838** [comment]
It's true that timestamp for the forecast is shown in your local time not in location's local time (Its subjective actually which one to show). But I believe dynamic background reflects the condition of the selected location, If it is not so can you pls share some screenshots.
